app/Http/Controllers/PermissionController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Common\GlobalVariable;
+use App\Models\Permission;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Gate;
+use Spatie\Permission\Models\Role as ModelsRole;
+
+class PermissionController extends Controller
+{
+    public $model = Permission::class;
+    protected $guard = 'web';
+
+    public function assignPermissionToRole(Request $request)
+    {
+        $role_id = $request->get('role_id');
```

Comment:
missing validation, implementation should at Model, not Controller, should not use transaction if not required